lib/connectors/base/connector.rb

```diff
@@ -33,6 +34,12 @@ def service_type
       def configurable_fields
         {}
       end
+
+      # connector-specific configuration that comes from the local yaml config file
+      # should be stored under the section that has the same name as the connector service_type
+      def local_configuration
+        App::Config[service_type] || {}
```

Comment:
I believe it should be passed as a parameter into the constructor to avoid having a `App::Config` usage outside of `App` module as we've discussed previously

Reply:
I think this way might be acceptable too since it's in the base class, so there's supposed to just be one usage? It looks a bit cleaner to me. Yeah we restrict the user a little because we're determining the way the settings should be stored, but in the end, isn't it better than invent a new wheel for every connector?
Or do you mean reading and passing these directly on `Connector.new`? As in, in the Registry?

Reply:
> Or do you mean reading and passing these directly on Connector.new? As in, in the Registry?

Yup, this part. The framework that creates the connector with `Connector.new` can read the config and pass it inside the connector, I feel it's a bit cleaner this way

Reply:
Agreed. Refactoring.